arviz/data/inference_data.py

```diff
@@ -589,6 +593,103 @@ def to_dataframe(
             (dfs,) = dfs.values()
         return dfs
 
+    def to_zarr(self, store=None):
+        """Convert InferenceData to a :class:`zarr.hierarchy.group`.
+
+        The zarr storage is using the same group names as the InferenceData.
+
+        Raises TypeError if no valid store is found.
```

Comment:
Maybe we should use the `Raises` section? See https://numpydoc.readthedocs.io/en/latest/format.html#sections point 9 for an example. We can then also add the ImportError to be extra clear on the need to install zarr independently